app/assets/stylesheets/projects.css.scss

```diff
@@ -27,3 +27,21 @@ h5 span.favorite {
 .project_links {
   margin-bottom: 6%;
 }
+
+.tag_list {
+  border-radius: 5px;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.